tutorials/cookbook.md

```diff
@@ -10,9 +10,9 @@ import IOT from "versionContent/_partials/_cookbook-iot.mdx";
 
 
 
-# Timescale community cookbook
+# TigerData community cookbook
 
-This page contains suggestions from the [TimescaleDB Community](https://timescaledb.slack.com/) about how to resolve 
```

Comment:
Gotta decide which community it is - tigerdata or timescaledb

Reply:
It is already used for Cloud as well as timescaledb. I'll ask if we will update the name of the timescaledb community to TigerData.